concent_api/core/tests/test_unit_validation.py

```diff
@@ -49,7 +51,7 @@
 (DIFFERENT_REQUESTOR_PRIV_ETH_KEY, DIFFERENT_REQUESTOR_PUB_ETH_KEY) = generate_priv_and_pub_eth_account_key()
 
 
-class TestValidateGolemMessageSubtaskResultsRejected(TestCase):
+class TestValidateGolemMessageSubtaskResultsRejected(unittest.TestCase):
```

Comment:
why is it changed?

Reply:
In this module both `unittest.TestCase` and `django.test.TestCase` are used.